src/components/TodoContainer.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+import React from 'react';
+import TodosList from './TodosList'
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+// import uuid from "uuid";
+import { v4 as uuidv4 } from "uuid";
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        // id: uuid.v4(),
+        id: uuidv4(),
+        title: "Setup development environment",
+        completed: true,
+      },
+      {
+        // id: uuid.v4(),
+        id: uuidv4(),
+        title: "Develop website and add content",
+        completed: false,
+      },
+      {
+        // id: uuid.v4(),
+        id: uuidv4(),
+        title: "Deploy to live server",
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    this.setState({
+      todos: this.state.todos.map((todo) => {
+        if (todo.id === id) {
+          todo.completed = !todo.completed;
+        }
+        return todo;
+      }),
+    });
+  };
+
+  delTodo = (id) => {
+    this.setState({
+      todos: [
+        ...this.state.todos.filter((todo) => {
+          return todo.id !== id;
+        }),
+      ],
+    });
+  };
+
+  addTodoItem = (title) => {
+    const newTodo = {
+      // id: uuid.v4(),
+      id: uuidv4(),
+      title: title,
+      completed: false,
+    };
+    this.setState({
+      todos: [...this.state.todos, newTodo],
```

Comment:
:slightly_smiling_face: Nice use of spreaders

Reply:
thankk youuuuu 😊